model/trace/gen-ai.yaml

```diff
@@ -0,0 +1,63 @@
+groups:
+  - id: gen_ai.llm.request
+    type: span
+    brief: >
+      A request to an LLM is modeled as a span in a trace. The span name should be a low cardinality value representing the request made to an LLM, like the name of the API endpoint being called.
+    attributes:
+      - ref: gen_ai.llm.system
+        requirement_level: recommended
+        note: >
+          The name of the LLM foundation model vendor, if applicable. If not using a vendor-supplied model, this field is left blank.
```

Comment:
```suggestion
          The name of the LLM foundation model vendor, if applicable. If not using a vendor-supplied model, provide a custom friendly name, such as a name of the company or project. If the instrumetnation reports any attributes specific to a custom model, the value provided in the `gen_ai.llm.system` SHOULD match the custom attribute namespace segment. For example, if `gen_ai.llm.system` is set to `the_best_llm`, custom attributes should be added in the `gen_ai.llm.the_best_llm.*` namespace.
          If none of above options apply, the instrumentation should set `_OTHER`.
```

`_OTHER` is a magic word we use in other places in OTel. If you have strong feelings about it, we can consider `other`, `unknown`, etc